diff -ur polkit-122.orig/data/meson.build polkit-122/data/meson.build
--- polkit-122.orig/data/meson.build	2022-10-26 14:12:24.000000000 +0200
+++ polkit-122/data/meson.build	2023-07-22 13:01:22.590545340 +0200
@@ -26,7 +26,7 @@
   )
 endif
 
-if enable_logind
+if session_tracking == 'libsystemd-login'
   configure_file(
     input: 'polkit.service.in',
     output: '@BASENAME@',
diff -ur polkit-122.orig/meson.build polkit-122/meson.build
--- polkit-122.orig/meson.build	2022-10-26 14:12:24.000000000 +0200
+++ polkit-122/meson.build	2023-07-22 13:00:31.328819850 +0200
@@ -204,7 +204,7 @@
 
   # systemd unit / service files
   systemd_systemdsystemunitdir = get_option('systemdsystemunitdir')
-  if systemd_systemdsystemunitdir == ''
+  if systemd_systemdsystemunitdir == '' and session_tracking == 'libsystemd-login'
     systemd_dep = dependency('systemd', not_found_message: 'systemd required but not found, please provide a valid systemd user unit dir or disable it')
     # FIXME: systemd.pc file does not use variables with relative paths, so `define_variable` cannot be used
     systemd_systemdsystemunitdir = systemd_dep.get_pkgconfig_variable('systemdsystemunitdir')
