--- polkit-0.112/configure.ac~	2014-12-29 22:40:04.000000000 +0000
+++ polkit-0.112/configure.ac	2014-12-29 22:41:01.765503630 +0000
@@ -129,16 +129,16 @@
 AC_DEFINE([GLIB_VERSION_MAX_ALLOWED], [G_ENCODE_VERSION(2,34)],
         [Notify us when we'll need to transition away from g_type_init()])
 
-AC_ARG_WITH(mozjs, AS_HELP_STRING([--with-mozjs=@<:@mozjs185/mozjs-17.0|auto@:>@],
+AC_ARG_WITH(mozjs, AS_HELP_STRING([--with-mozjs=@<:@mozjs187/mozjs-17.0|auto@:>@],
 		                  [Specify version of Spidermonkey to use]),,
             with_mozjs=auto)
 AS_IF([test x${with_mozjs} != xauto], [
   PKG_CHECK_MODULES(LIBJS, ${with_mozjs})
 ], [
-  PKG_CHECK_MODULES(LIBJS, [mozjs185], have_mozjs185=yes, have_mozjs185=no)
-  AS_IF([test x${have_mozjs185} = xno], [
+  PKG_CHECK_MODULES(LIBJS, [mozjs187], have_mozjs187=yes, have_mozjs187=no)
+  AS_IF([test x${have_mozjs187} = xno], [
   PKG_CHECK_MODULES(LIBJS, [mozjs-17.0], have_mozjs17=yes,
-   [AC_MSG_ERROR([Could not find mozjs185 or mozjs-17.0; see http://ftp.mozilla.org/pub/mozilla.org/js/])])
+   [AC_MSG_ERROR([Could not find mozjs187 or mozjs-17.0; see http://ftp.mozilla.org/pub/mozilla.org/js/])])
   ])
 ])
 AC_SUBST(LIBJS_CFLAGS)
