--- polkit-0.110/configure.ac.orig	2013-01-12 13:32:07.257259065 +0100
+++ polkit-0.110/configure.ac	2013-01-12 13:32:54.047258091 +0100
@@ -3,7 +3,7 @@
 AC_PREREQ(2.59c)
 AC_INIT(polkit, 0.110 http://lists.freedesktop.org/mailman/listinfo/polkit-devel)
 AM_INIT_AUTOMAKE(polkit, 0.110)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 AM_MAINTAINER_MODE
 
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
