--- polkit-0.114/src/polkitbackend/polkitbackendjsauthority.cpp.orig	2018-04-12 21:26:58.848880061 +0200
+++ polkit-0.114/src/polkitbackend/polkitbackendjsauthority.cpp	2018-04-12 22:06:53.862186041 +0200
@@ -1296,7 +1296,7 @@
   JS::CallArgs args = JS::CallArgsFromVp (argc, vp);
 
   s = JS_EncodeString (cx, args[0].toString ());
-  JS_ReportWarningUTF8 (cx, s);
+  JS_ReportWarningUTF8 (cx, "%s", s);
   JS_free (cx, s);
 
   ret = true;
